examples/specs/anscombe.json

```diff
@@ -0,0 +1,69 @@
+{
+  "description": "Anscombe's Quarter (https://en.wikipedia.org/wiki/Anscombe%27s_quartet)",
+  "data": {
+    "values": [
+      {"Series":"I", "X":10.0, "Y":8.04},
```

Comment:
I think it's worth putting this dataset in https://github.com/vega/vega-datasets